packages/api-reference-editor/tsconfig.json

```diff
@@ -1,9 +1,13 @@
 {
   "extends": "../../tsconfig.json",
+  "include": ["src"],
+  "exclude": ["dist", "vite.config.ts", "node_modules"],
```

Comment:
nice 😮‍💨

Reply:
actually, we should include the vite.config.ts, excluding it happened just by accident (copied from api-reference) :D